src/qt/guiutil.cpp

```diff
@@ -175,7 +175,8 @@ bool parseBitcoinURI(QString uri, SendCoinsRecipient *out)
 
 QString formatBitcoinURI(const SendCoinsRecipient &info)
 {
-    QString ret = QString("bitcoin:%1").arg(info.address);
+    bool bech_32 = (info.address.startsWith("bc1") || info.address.startsWith("tb1"));
```

Comment:
I don't like hardcoding this here—don't we have constants to use somewhere?

Reply:
I looked around and couldn't find any constants, I could add them unless someone else knows if some exist already.

Reply:
there's nothing on `ChainParams` ?

Reply:
Yeah, you can test whether the address starts with `params.Bech32HRP() + "1"`.

Reply:
`params` isn't available in this scope

Reply:
You can get the global params object by calling `Params()`.